docs/QuickStart.md

```diff
@@ -185,15 +72,15 @@ Once you have your Function App you need to configure it for use with Azure SQL
     (<i>Note: when pasting in the connection string, you will need to replace part of the connection string where it says '{your_password}' with your Azure SQL Server password</i>)
    </details>
 
-1. Open the generated `local.settings.json` file and in the `Values` section verify you have the below. If not, add the below and replace `{connection_string}` with the your connection string from the previous step:
+3. Open the generated `local.settings.json` file and in the `Values` section verify you have the below. If not, add the below and replace `{connection_string}` with the your connection string from the previous step:
 
     ```json
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "AzureWebJobsDashboard": "UseDevelopmentStorage=true",
     "SqlConnectionString": "{connection_string}"
     ```
 
-1. Verify your `host.json` looks like the below:
+4. Verify your `host.json` looks like the below:
```

Comment:
While you're here - could you remove this? It isn't necessary since we aren't actually changing anything and we don't want to confuse people if the default host.json generated changes.